modules/javafx.controls/src/test/java/test/javafx/scene/control/TreeTableRowTest.java

```diff
@@ -820,4 +834,71 @@ public class TreeTableRowTest {
         TreeTableRow cell = new TreeTableRow();
         cell.setSkin(new TreeTableRowSkin(cell));
     }
+
+    /** TreeTableView with cell selection enabled should not select TreeTableRows */
+    @Test
+    public void test_TreeTableView_jdk_8292353_select_all() {
```

Comment:
This test passes with or without the code changes.
If test intention is to increase coverage, then the comment `// JDK-8292353 failure` needs to be removed. If not, the test needs some correction.

Reply:
similarly, this test also fails without code changes:

```
TreeTableRowTest > test_TreeTableView_jdk_8292353_select_all FAILED
    org.opentest4j.AssertionFailedError: expected: <false> but was: <true>
        at app//org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:55)
        at app//org.junit.jupiter.api.AssertFalse.assertFalse(AssertFalse.java:40)
        at app//org.junit.jupiter.api.AssertFalse.assertFalse(AssertFalse.java:35)
        at app//org.junit.jupiter.api.Assertions.assertFalse(Assertions.java:227)
        at app//test.javafx.scene.control.TreeTableRowTest.test_TreeTableView_jdk_8292353_select_all(TreeTableRowTest.java:870)
```

Reply:
A clean build and run behaves correctly. This test fails without code changes and passes with them.
Sorry for the confusion!